src/sugar3/graphics/toolbutton.py

```diff
@@ -58,6 +62,22 @@ def setup_accelerator(tool_button):
 
 
 class ToolButton(Gtk.ToolButton):
+    '''
+    UI for toolbutton.
+    A ToolButton is a ToolItem having an icon, a tooltip palette,
+    and an accelerator.
+    Use ToolButton() to create a new ToolButton.
+    Fuctions:
+        ToolButton()
```

Comment:
You don't need to document the functions, this documentation is shown next to the custructor (`__init__`) anyway.